src/main/java/tornadofx/Component.kt

```diff
@@ -712,57 +712,54 @@ abstract class UIComponent(viewTitle: String? = "", icon: Node? = null) : Compon
     @JvmOverloads
     fun openModal(stageStyle: StageStyle = StageStyle.DECORATED, modality: Modality = Modality.APPLICATION_MODAL, escapeClosesWindow: Boolean = true, owner: Window? = currentWindow, block: Boolean = false, resizable: Boolean? = null): Stage? {
         if (modalStage == null) {
-            if (getRootWrapper() !is Parent) {
-                throw IllegalArgumentException("Only Parent Fragments can be opened in a Modal")
-            } else {
-                modalStage = Stage(stageStyle)
-                // modalStage needs to be set before this code to make close() work in blocking mode
-                with(modalStage!!) {
-                    if (resizable != null) isResizable = resizable
-                    titleProperty().bind(titleProperty)
-                    initModality(modality)
-                    if (owner != null) initOwner(owner)
-
-                    if (getRootWrapper().scene != null) {
-                        scene = getRootWrapper().scene
-                        this@UIComponent.properties["tornadofx.scene"] = getRootWrapper().scene
-                    } else {
-                        Scene(getRootWrapper()).apply {
-                            if (escapeClosesWindow) {
-                                addEventFilter(KeyEvent.KEY_PRESSED) {
-                                    if (it.code == KeyCode.ESCAPE)
-                                        close()
-                                }
+            require(getRootWrapper() !is Parent) { "Only Parent Fragments can be opened in a Modal" }
```

Comment:
Shouldn't this be `require(getRootWrapper() is Parent)`? The `!is` seems backwards.

Reply:
YES